src/devices/schneider_electric.ts

```diff
@@ -475,6 +483,21 @@ const definitions: Definition[] = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['CHFAN/SWITCH/1'],
+        model: '41ECSFWMZ-VW',
+        vendor: 'Schneider Electric',
+        description: 'Wiser 40/300-Series Module AC Fan Controller',
+        fromZigbee: [fz.on_off, fz.fan],
+        toZigbee: [tzLocal.fan_mode, tz.on_off],
```

Comment:
What does the `tz.on_off` do here? I think only `fan_mode` should be here (same for `fromZigbee`)

Reply:
You're right! I thought this was required from when I first added this integration but seems`fan_mode` has all the correct endpoint/exposed attributes.

I removed the `onOff` from the reporting as well as this seems to be part of the `fanMode` binding? I tested it on my fan and all the data appears correctly and fan works as expected.

Reply:
Also added `off` to the array just so you can get the option for it in the drop down box within the Zigbee UI, even though the `off` option is defaulted into the array in the backend.